### PR TITLE
feature/Filter-path-by-regex: added new filter for filtering by path …

### PR DESCRIPTION
**Pull request by roxana-muresan**
…regex.

### CODE REVIEW

File: api/src/main/java/de/valtech/aecu/api/groovy/console/bindings/filters/FilterByPathRegex.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package de.valtech.aecu.api.groovy.console.bindings.filters;
+
+import org.apache.sling.api.resource.Resource;
+
+import javax.annotation.Nonnull;
+
+public class FilterByPathRegex implements FilterBy {
```
Comment (gruberrolandvaltech) [inline api/src/main/java/de/valtech/aecu/api/groovy/console/bindings/filters/FilterByPathRegex.java]: Class level JavaDoc

Comment (roxana-muresan) [reply api/src/main/java/de/valtech/aecu/api/groovy/console/bindings/filters/FilterByPathRegex.java]: Done

File: Readme.md [hunk 2]
```diff
@@ -246,7 +244,20 @@ println aecu.contentUpgradeBuilder()
         .run()
 ```
 
-<a name="jmx"></a>
+### Filter by Node Path regex
```
Comment (gruberrolandvaltech) [inline Readme.md]: Filter by Node Path regex -> Filter by Node Path
Fits better to other headlines and we might add more methods.

File: Readme.md [hunk 3]
```diff
@@ -246,7 +244,20 @@ println aecu.contentUpgradeBuilder()
...
+
+Nodes can also be filter by their path using a regular expression.
```
Comment (gruberrolandvaltech) [inline Readme.md]: be filter -> be filtered

File: Readme.md [hunk 4]
```diff
@@ -246,7 +244,20 @@ println aecu.contentUpgradeBuilder()
...
+
+* filterByPathRegex(String regex): process nodes who's path matches the given regular expression
```
Comment (gruberrolandvaltech) [inline Readme.md]: who's -> whose

Comment (roxana-muresan) [reply Readme.md]: 😁

File: HISTORY [hunk 5]
```diff
@@ -3,6 +3,7 @@
     - Added SQL2 collection method
     - Page actions: manage tags
     - Resource actions: rename resource
+    - New Filter: filter by path regex
```
Comment (gruberrolandvaltech) [inline HISTORY]: New Filter: filter by path regex -> New **filter**: filter by path regex

File: Readme.md (hunk 3 above)
Comment (roxana-muresan) [reply Readme.md]: 😁

**Pull request merged**